mklove/modules/configure.libsasl2

```diff
@@ -7,23 +7,26 @@
 #
 #
 # And then call the following function from the correct place/order in checks:
-#   mkl_check libsasl2 [<action>]
+#   mkl_check libsasl2
 #
 
-mkl_toggle_option "Feature" ENABLE_GSSAPI "--enable-gssapi" "Enable SASL GSSAPI support with Cyrus libsasl2" "y"
+mkl_toggle_option "Feature" ENABLE_GSSAPI "--enable-gssapi" "Enable SASL GSSAPI support with Cyrus libsasl2" "d"
```

Comment:
I'm guessing this means the --help output will show "(d)" as the default, which doesn't mean much to the user.
We should probably have --help show "try", "if available", or somesuch as the default string for "d".

Reply:
Ah, sure. I didn't see an obvious way to plumb that through `mkl_toggle_option` and `mkl_option`, so... instead I just changed the default from "d" to "try". That seems to have the desired effect and doesn't require any additional code changes. What do you think?